# These are offsets from the calibrate 0 position.  Typically between -60 to 60.  0 is treated as a special case, to cut power to the servo
int8 right_front
int8 right_back
int8 left_front
int8 left_back
int8 arm_updown
int8 arm_leftright